rubin_sim/maf/metrics/summary_metrics.py

```diff
@@ -205,31 +205,61 @@ def run(self, data_slice, slice_point=None):
             return result
 
 
+class FskyMetric(BaseMetric):
```

Comment:
How is this different than the AreaThresholdMetric? And could we just modify the existing metric to add the extra capabilities, rather than introducing an entirely new version? (I would like to keep similar but slightly different code to a minimum, if possible).

Reply:
I'm also wondering about pixels with hp.UNSEEN or np.nan .. these *should* be getting masked in the metric_values masked array I would think?

Reply:
thanks for pointing out AreaThresholdMetric, I hadn't seen it, but I now figured out how to use it, and thus I have removed the FskyMetric (see new commits). I have made two very small changes to AreaThresholdMetric though.